src/style.css

```diff
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
+
+.page {
+  color: #3d4e61;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+.page__form {
+  margin-top: 20px;
+}
+
+.page__body {
+  margin-block: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.search-bar {
+  box-sizing: border-box;
+  width: 100%;
+  font-family: inherit;
+  color: inherit;
+
+  background-color: #fff;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 4px;
```

Comment:
You've used `box-sizing: border-box;` which is good as it prevents the padding and border from affecting the actual dimensions of the boxes.